java/system-test/system-test-drivers/src/test/java/sleeper/systemtest/drivers/statestore/AwsStateStoreCommitterDriverIT.java

```diff
@@ -129,7 +130,10 @@ private String getMessageGroupId(Message message) {
     }
 
     private StateStoreCommitRequest readCommitRequest(Message message) {
-        return new StateStoreCommitRequestDeserialiser(instance.getTablePropertiesProvider())
+        LoadS3ObjectFromDataBucket noLoadFromS3 = key -> {
+            throw new UnsupportedOperationException("Did not expect message held in S3");
```

Comment:
Language for message here sounds wrong, not able to think of better suggestion though

Reply:
I've just made it actually load from S3, thanks.